src/views/mentorship/mentorship-hero/ui/mentorship-hero.tsx

```diff
@@ -0,0 +1,78 @@
+import classNames from 'classnames/bind';
+import Image, { StaticImageData } from 'next/image';
+import { PAGE_NAMES } from '@/shared/constants';
+import { LinkCustom } from '@/shared/ui/link-custom';
+import { MainTitle } from '@/shared/ui/main-title';
+import { Subtitle } from '@/shared/ui/subtitle';
+import { heroPageData, mentorsRegisterData } from 'data';
+
+import styles from './mentorship-hero.module.scss';
+
+export const cx = classNames.bind(styles);
+
+export type MentorshipHeroProps = {
+  mainTitle: string;
+  widgetTitle: string;
+  subTitle: string[] | [];
+  heroImageSrc?: StaticImageData;
+  imageAltText: string;
+};
```

Comment:
This type contains the word props, but it is not used for that purpose

Reply:
Deleted